src/Blazored.Toast/BlazoredToasts.razor.cs

```diff
@@ -8,18 +8,21 @@
 
 namespace Blazored.Toast
 {
+    public enum IconType { FontAwesome, Material };
+
     public partial class BlazoredToasts
     {
         [Inject] private IToastService ToastService { get; set; }
 
+        [Parameter] public IconType IconType { get; set; } = IconType.FontAwesome;
```

Comment:
Should we default this? I'm not sure we should assume FontAwesome is the default icon choice. Maybe we should have it as a required parameter with a null check.

Reply:
Before I changed the names of the parameters this kept everything working as it did originally (It was done with a different parameter but the idea was the same). Your call, I'm pretty agnostic on this.

Reply:
I think now we've changed the API we might as well go all in. By making the `IconType` mandatory it makes things consistent.

Reply:
Done; I had to add the errorUI html to the html startup pages in both examples. In SSB it displays the error as expected. In CSB the error briefly flashes and then the app continues and gets rendered. The null IconType prevents the icons from displaying. I observed the continuation without the error message looking at the console in CSB (that's what made me add the errorUI). I think that this is a CSB issue with wasm. I will raise an issue on the Asp.Net repository if you don't have any special insight.

Reply:
For you to look at this just take the IconType parameter out of the client side example.